Problem With Commit Info Guide

The guide at [guides.micronaut.io/latest/adding-commit-info-gradle-java.html](https://guides.micronaut.io/latest/adding-commit-info-gradle-java.html) provides a link to download the completed example from [guides.micronaut.io/latest/adding-commit-info-gradle-java.zip](https://guides.micronaut.io/latest/adding-commit-info-gradle-java.zip).

The project in that .zip file will not build because the project is using `com.gorylenko.gradle-git-properties` but the project in the .zip file isn't a git repo.

```
$ ./gradlew run
> Task :generateGitProperties FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateGitProperties'.
> No Git repository found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 829ms
2 actionable tasks: 1 executed, 1 up-to-date
```

> is this solved?

@sdelamo I don't think so.

```
% curl --output adding-commit-info-gradle-java.zip https://guides.micronaut.io/latest/adding-commit-info-gradle-java.zip && unzip  adding-commit-info-gradle-java.zip -d adding-commit-info && cd adding-commit-info && ./gradlew run
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 68806  100 68806    0     0   298k      0 --:--:-- --:--:-- --:--:--  305k
Archive:  adding-commit-info-gradle-java.zip
  inflating: adding-commit-info/gradlew.bat  
  inflating: adding-commit-info/gradle.properties  
  inflating: adding-commit-info/micronaut-cli.yml  
   creating: adding-commit-info/gradle/
   creating: adding-commit-info/gradle/wrapper/
  inflating: adding-commit-info/gradle/wrapper/gradle-wrapper.jar  
  inflating: adding-commit-info/gradle/wrapper/gradle-wrapper.properties  
  inflating: adding-commit-info/build.gradle  
   creating: adding-commit-info/src/
   creating: adding-commit-info/src/test/
   creating: adding-commit-info/src/test/java/
   creating: adding-commit-info/src/test/java/example/
   creating: adding-commit-info/src/test/java/example/micronaut/
  inflating: adding-commit-info/src/test/java/example/micronaut/InfoTest.java  
  inflating: adding-commit-info/src/test/java/example/micronaut/DefaultTest.java  
   creating: adding-commit-info/src/main/
   creating: adding-commit-info/src/main/resources/
   creating: adding-commit-info/src/main/resources/META-INF/
   creating: adding-commit-info/src/main/resources/META-INF/native-image/
  inflating: adding-commit-info/src/main/resources/META-INF/native-image/resource-config.json  
  inflating: adding-commit-info/src/main/resources/logback.xml  
  inflating: adding-commit-info/src/main/resources/application.yml  
   creating: adding-commit-info/src/main/java/
   creating: adding-commit-info/src/main/java/example/
   creating: adding-commit-info/src/main/java/example/micronaut/
  inflating: adding-commit-info/src/main/java/example/micronaut/Application.java  
  inflating: adding-commit-info/settings.gradle  
  inflating: adding-commit-info/README.md  
  inflating: adding-commit-info/gradlew  
> Task :generateGitProperties FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateGitProperties'.
> Error while evaluating property 'generatedProperties' of task ':generateGitProperties'.
   > No Git repository found.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 655ms
2 actionable tasks: 2 executed
```

Does that build succeed when you try?

--- a/guides/adding-commit-info/adding-commit-info.adoc
+++ b/guides/adding-commit-info/adding-commit-info.adoc
@@ -30,7 +30,7 @@ The project aims to demonstrate how to provide Git commit information to the
 cd micronautguide
 git init
 git add .
-git commit -a "Initial project"
+git commit -am "Initial project"
 ----
 
 == Management

